models/multimeter.h

```diff
@@ -110,6 +112,19 @@ fail if carried out in the wrong direction, i.e., trying to connect the
    ``record_from`` property is already set to record the variable ``V_m``
    from the neurons it is connected to.
 
+.. include:: ../models/recording_device.rst
+
+.. glossary::
+
+ record_from
+   A list (default: `[]`) of parameters and state variables to sample
+   from the nodes, the multimeter is connected to. Potential
+   recordables are given in the corresponding model documentation.
+
+ interval
+   A float (default: `1.0`) specifying the interval in ms, at which
+   data is collected from the nodes, the multimeter is connected to.
+
 EndUserDocs */
```

Comment:
Add a See-also section to actually refer to other recorders
```suggestion

See also
++++++++

EndUserDocs */
```